packages/@o3r/pipeline/schematics/ng-add/templates/github/__dot__github/workflows/main.yml

```diff
@@ -0,0 +1,54 @@
+name: Main CI
+
+on:
+  push:
+    branches:
+      - main
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
+      - 'release/*'
+
+jobs:
+  build:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Build
+        run: <%= packageManager %> run build
+
+  code-checks:
+    runs-on: <%= runner %>
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: Unit tests
+        run: <%= packageManager %> run test
+      - name: Audit
+        run: <% if (packageManager === 'yarn') {%>yarn <%}%>npm audit
+      - name: Lint
+        run: <%= packageManager %> run lint:ci
+  version:
+    permissions:
+      contents: write
+    runs-on: <%= runner %>
+    outputs:
+      nextVersionTag: ${{ steps.new-version.outputs.nextVersionTag }}
+    steps:
+      - uses: actions/checkout@692973e3d937129bcbf40652eb9f2f61becf3332 # v4.1.7
+      - uses: ./.github/actions/setup
+      - name: New version
+        if: github.event_name != 'merge_group'
+        id: new-version
+        uses: AmadeusITGroup/otter/tools/github-actions/new-version@main
```

Comment:
Should we recommend to use the `main` or a released version ?

Reply:
We may want to configure Renovate to upgrade to upgrade the commit number with the latest from `release/*` branches